.github/workflows/relay_e2e_health.yml

```diff
@@ -0,0 +1,55 @@
+name: Relay e2e health check
+on:
+  schedule:
+    - cron: '0 8 * * *'

```

Comment:
Nit: Maybe this should run at a slightly different time than the playwright tests?